python/test/CMakeLists.txt

```diff
@@ -738,6 +738,16 @@ ot_pyinstallcheck_test (ANCOVA_std)
 ot_pyinstallcheck_test (FAST_std)
 ot_pyinstallcheck_test (SobolIndicesAlgorithm_std)
 ot_pyinstallcheck_test (SobolIndicesExperiment_std)
+ot_pyinstallcheck_test (HSICStat_std)
+if (NUMPY_FOUND)
+  ot_pyinstallcheck_test (HSICEstimator_std)
+  ot_pyinstallcheck_test (HSICEstimatorConditionalSensitivity_std)
+  ot_pyinstallcheck_test (HSICEstimatorTargetSensitivity_std)
+  ot_pyinstallcheck_test (HSICEstimatorGlobalSensitivity_std)
+endif ()
```

Comment:
it seems numpy is just needed for np.inf here, using the built-in float('inf') would allow to always test these (mingw/windows)

Reply:
OK, thank you. I change it.